packages/blade/src/components/List/ListItemCode.tsx

```diff
@@ -0,0 +1,14 @@
+import type { CodeProps } from '../Typography';
+import { Code } from '../Typography';
+import { useListContext } from './ListContext';
+import { listItemCodeSize } from './listTokens';
+
+type ListItemCodeProps = Exclude<CodeProps, 'size' | 'variant' | 'isDisabled'>;
```

Comment:
why exclude? we shall pass all the necessary props for code right?

Reply:
Copy paste issue. Code doesn't have variant or isDisabled. Removed it